src/main/resources/application.yml

```diff
@@ -203,3 +203,17 @@ logging:
     com:
       jcraft:
         jsch: WARN
+
+email:
+  client:
+    sender: noreply_erouser@erop.ierds.uk
+    allow-list-enabled: false
+    allow-list-domains:
+  content:
+    portal-base-url: https://erop.ierds.uk
```

Comment:
Did you mean to add `EMAIL_CONTENT_PORTAL_BASE_URL` in this? If yes, it also needs to be added to README please